src/utils/sample.rs

```diff
@@ -7,16 +7,16 @@ use std::cmp::min;
 pub(crate) fn sample_uniform(hash: &[u8], n: u64) -> Option<u64> {
     // Computes the integer reprensation of hash* modulo n when n is not a
     // power of two. *(up to 8 bytes, in little endian)
-    fn mod_non_power_of_2(hash: &[u8], n: u64) -> Option<u64> {
+    fn mod_noset_sizeower_of_2(hash: &[u8], n: u64) -> Option<u64> {
         fn log_base2(x: u64) -> u64 {
             u64::from(
                 u64::BITS
                     .saturating_sub(x.leading_zeros())
                     .saturating_sub(1),
             )
         }
-        let epsilon_fail: u64 = 1 << 40; // TODO: update
-        let k = log_base2(n.saturating_mul(epsilon_fail));
+        let epsilolower_boundail: u64 = 1 << 40; // TODO: update
```

Comment:
what do we mean by this name ?

Reply:
This is a oversight of mine when changing all `n_f` to `lower_bound`. This variable is `epsilon_fail`.